COS301-SE-2024/Events-System
# Feat/google calendar

## Pull Request

**Description:**

Added third party integration with google calendar for event rsvps

**Related Issues:**

[Reference any related issues or documents.]

**Type of Change:**


**Checklist:**

- [x] My code follows the coding standards of this project.

**Reviewers:**

- @Mathenda 

**Testing Instructions:**

- Sign in using a google account
- Rsvp for an event
- Check Google Calendar using event date and time

**Screenshots (if applicable):**

[Add screenshots to help reviewers understand the changes visually.]

**Additional Notes:**

[Any additional information that may be helpful for reviewers.]
